apps/src/gamelab/spritelab/spriteUtils.js

```diff
@@ -0,0 +1,22 @@
+export var spriteId = 0;
+export var nativeSpriteMap = {};
+
+function allSpritesWithAnimation(animation) {
+  let group = [];
+  Object.keys(nativeSpriteMap).forEach(spriteId => {
+    if (nativeSpriteMap[spriteId].getAnimationLabel() === animation) {
+      group.push(nativeSpriteMap[spriteId]);
+    }
+  });
+  return group;
+}
+
+export function singleOrGroup(spriteOrGroup) {
```

Comment:
util function to handle whether a function was called with a named sprite or a costume group.
Returns a list of sprites (containing either just the named sprite or all sprites with the specified costume)